app/helpers/requests_helper.rb

```diff
@@ -0,0 +1,24 @@
+module RequestsHelper
+  def status_request_options_for_select
+    [[t("requests.all"), ""],
+    [t("requests.approved"), "approved"],
+    [t("requests.pending"), "pending"],
+    [t("requests.rejected"), "rejected"],
+    [t("requests.canceled"), "cancel"]]
```

Comment:
Chỗ này mình dùng Request.status để lấy ra toàn bộ status rồi dùng vòng lặp để xử lý nhé